packages/ckeditor5-clipboard/src/dragdropexperimental.ts

```diff
@@ -690,3 +685,44 @@ function findDraggableWidget( target: ViewElement ): ViewElement | null {
 
 	return null;
 }
+
+/**
+ * Recursively checks if all provided elements have the same parent and if that parent doesn't have
+ * any other children. If that's the case, it returns range including this parent. Otherwise,
+ * it returns only the range from first to last element.
+ *
+ * Example:
+ *
+ * <blockQuote>
+ *   <paragraph>[Test 1</paragraph>
+ *   <paragraph>Test 2</paragraph>
+ *   <paragraph>Test 3]</paragraph>
+ * <blockQuote>
+ *
+ * Because all elements inside the `blockQuote` are selected, the range includes the `blockQuote` too.
+ * If only first and second paragraphs would be selected, the range would not include it.
+ */
+function getRangeExtendedToLargestFullySelectedParent( model: Model, elements: Array<Element> ): Range {
```

Comment:
To the `getRangeExtendedToLargestFullySelectedParent` we pass **blocks**, so maybe it could be good name for this argument instead **elements**, what do you think? To be honest I would call it **selectedBlocks**.